platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbPaths.java

```diff
@@ -43,10 +43,10 @@ public MerkleDbPaths(final Path storageDir) {
         metadataFile = storageDir.resolve("metadata.jdbm");
         pathToDiskLocationInternalNodesFile = storageDir.resolve("pathToDiskLocationInternalNodes.ll");
         pathToDiskLocationLeafNodesFile = storageDir.resolve("pathToDiskLocationLeafNodes.ll");
-        internalHashStoreRamFile = storageDir.resolve("internalHashStoreRam.hl");
-        internalHashStoreDiskDirectory = storageDir.resolve("internalHashStoreDisk");
+        hashStoreRamFile = storageDir.resolve("internalHashStoreRam.hl");
+        hashStoreDiskDirectory = storageDir.resolve("internalHashStoreDisk");
         longKeyToPathFile = storageDir.resolve("longKeyToPath.ll");
         objectKeyToPathDirectory = storageDir.resolve("objectKeyToPath");
-        pathToHashKeyValueDirectory = storageDir.resolve("pathToHashKeyValue");
+        pathToKeyValueDirectory = storageDir.resolve("pathToHashKeyValue");
```

Comment:
It would be great to rename all these folders, too. Some time ago I implemented mechanism to change store file names (e.g. check "legacyStoreName" in MIDKVS constructor). Similar mechanism can hopefully be implemented for folder names. It would let use "hashStore" instead of "internalHashStoreDisk" and "leafStore" instead of "pathToHashKeyValue".

Reply:
I created a separate ticket - https://github.com/hashgraph/hedera-services/issues/6100 - to address this issue.